src/modules/data/blocks/ticket/selectors.js

```diff
@@ -578,6 +578,16 @@ export const getTicketProviders = () => {
 	return tickets.providers || [];
 };
 
+export const getDefaultProvider = () => {
```

Comment:
can we follow the naming convention here to stay consistent (`getDefaultTicketProvider`)